bin/music_genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_accessor :items, :name
+
+  def initialize(name)
+    @id = Random.rand(1..1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
```

Comment:
Good job on this method, please, don't forget to add the `add_gender` method to the `MusicAlbun` class according to the diagram.

Reply:
Thank you very much, I have done the corrections